jbpm/jbpm-flow/src/main/java/org/jbpm/workflow/instance/impl/WorkflowProcessInstanceImpl.java

```diff
@@ -1163,8 +1163,7 @@ public void setErrorState(NodeInstance nodeInstanceInError, Exception e) {
         this.nodeIdInError = nodeInstanceInError.getNodeDefinitionId();
         this.nodeInstanceIdInError = nodeInstanceInError.getId();
         this.errorCause = Optional.of(e);
-        Throwable rootException = getRootException(e);
-        this.errorMessage = rootException.getClass().getCanonicalName() + " - " + rootException.getMessage();
+        this.errorMessage = getRootException(e).getMessage();
```

Comment:
Previously if `getMessage` returned null, at least the concatenation with " - " converted to String, but not now.

```suggestion
        this.errorMessage = getRootException(e).getMessage() == null ? "" : getRootException(e).getMessage() ;
```